.github/workflows/create-github-release.yml

```diff
@@ -2,26 +2,26 @@ name: create-github-release
 on:
   pull_request:
     branches:
-      - main
+      - main-5
```

Comment:
We'll need to create a `main-5` branch, which we could do based off of `dev-5` right before we run the release branch creation job.